### PR TITLE
bug(badge): fix oval shape for api flow count badge, add tooltip

### PR DESCRIPTION
**Pull request by michael-coker**
fixex https://github.com/syndesisio/syndesis/issues/3601

### CODE REVIEW

File: app/ui/src/app/integration/list/list.component.scss [hunk 1]
```diff
@@ -16,14 +16,19 @@
 
     .icon {
       position: relative;
-      .badge {
+
+      .syn-badge {
         position: absolute;
         background: #ec7a08;
```
Comment (zregvart) [inline app/ui/src/app/integration/list/list.component.scss]: Any chance we can make this other color? I know that orange is the new black, but it screams like error to me, we can hope that it's me.

Comment (michael-coker) [reply app/ui/src/app/integration/list/list.component.scss]: @dongniwang what do you think?

Comment (dongniwang) [reply app/ui/src/app/integration/list/list.component.scss]: We’re going to gather user feedback on API provider and we can definitely get some feedback on the color of the badge. A more neutral color choice I can think of is perhaps blue? I’ll check with a visual design SME to get some suggestions.

Comment (michael-coker) [reply app/ui/src/app/integration/list/list.component.scss]: screen shot 2018-10-10 at 8 40 33 am
screen shot 2018-10-10 at 8 40 26 am
screen shot 2018-10-10 at 8 39 46 am

@dongniwang that sounds good to me. here's a screenshot of the dark and light blue, and one with the list row hover state (blue background)

File: app/ui/src/app/integration/list/list.component.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
           <span class="fa fa-angle-right"></span>
           <span class="icon">
             <img class="syn-icon-small" src="./../../assets/icons/multi-flow.connection.png" aria-hidden="true">
-            <span class="badge">{{ integration.flows.length }}</span>
+            <span class="syn-badge" tooltip="{{ integration.flows.length }} implemented flows" container="body" placement="right">{{ integration.flows.length }}</span>
```
Comment (riccardo-forina) [inline app/ui/src/app/integration/list/list.component.html]: `{{ integration.flows.length }}` will yield the number of flows, aka operations, coming from the spec. AFAIK there is no clear way to know the number of _implemented_ flows. @nicolaferraro is there any property we can use to figure out if an operation is implemented or not, but checking if the last step returns anything different from a 501?

Comment (nicolaferraro) [reply app/ui/src/app/integration/list/list.component.html]: We have talked about that in a call. I remember that 501 should be the "flag" that distinguish a implemented operation from a non-implemented one.

**Comment by gashcrumb** [comment]
Is this ready to get merged?

**Comment by michael-coker** [comment]
@gashcrumb - I believe @dongniwang is still working on how to present the badge.

**Comment by gashcrumb** [comment]
Is it worth maybe merging this one as the groundwork and then just open a PR with the final work later?

**Pull request merged**